buddyboss/buddyboss-platform
# #1574 - Fixed Load more photos issue

### All Submissions:


Fixes #1574 

### How to test the changes in this Pull Request:

1. Go to Profile -> Photos tab -> Album -> Add Photos tab
2. Click "Load More" button and see new photos are not loading

### Proof Screenshots or Video
https://prnt.sc/uqkicz

### Other information:
